src/Maths/math.constants.ts

```diff
@@ -3,11 +3,32 @@
  * @ignorenaming
  */
 export const ToGammaSpace = 1 / 2.2;
+
 /**
  * Constant used to convert a value to linear space
  * @ignorenaming
  */
 export const ToLinearSpace = 2.2;
+
+/**
+ * Constant Golden Ratio value in Babylon.js
+ * @ignorenaming
+ */
+ export const PHI = (1 + Math.sqrt(5)) / 2;
+
+ /**
+ * Constant one third of the square root of 3
+ * @ignorenaming
+ */
+  export const THIRDR3 = Math.sqrt(3) / 3;
+
+/**
+ * Constant used to define the isometric grid size in Babylon.js
+ * @ignorenaming
+ */
+ let IsoGridSize = 0.5;
+ export { IsoGridSize };

```

Comment:
Same comment here even more cause this file should only contain true constants. Also isoGridSize might be better off as an option with a default value in isoVector class ?

Reply:
Done